gprotorch/kernels/gnn_kernels/test_hu_iclr_2020.py

```diff
@@ -0,0 +1,357 @@
+"""
+Reformatted and cleaned up GNN modules from Hu et al.
+Strategies for Pre-training Graph Neural Networks. ICLR 2020
+(https://github.com/snap-stanford/pretrain-gnns)
+
+Author: Leo Klarner (https://github.com/leojklarner), April 2022
+"""
+
```

Comment:
Should we call the module test if it's not in the tests folder?

Reply:
Thanks, my bad!